Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime-Checkout-SAL-Scripts.rst

```diff
@@ -0,0 +1,111 @@
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-Daytime-Checkout-Daytime-Checkout:
+
+############################
+Daytime Checkout SAL Scripts
+############################
+
+To ensure the Auxiliary Telescope is fully operational and ready for nighttime use, 
+we have created a series of SAL Scripts that successively checkout each of the major components, 
+culminating in a full test of the system involving slewing to a target, tracking, and taking a test image with LATISS. 
+The goal of these scripts is to test the functionality of each component within the current software environment,
+and alert the daytime staff to any operational issues before going on-sky. 
+These tests are designed to be run using the script queue on LOVE during the daytime with an Observing Specialist 
+or other personnel available to verify their results. 
+
+The scripts are intended to be run in order, 
+but offer some flexibility in what state to leave the Auxiliary Telescope 
+(e.g. ready to take calibrations, ready to observe at night, or simply parked and shutdown). 
+They can also be run individually if a single component is undergoing maintenance 
+and requires a check of basic functionality. 
+
+None of the scripts require configuration. 
+
+.. Important::
+
+    The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. 
+    Before running, ensure that the area surrounding the telescope is and dome is clear for movement. 
+    You MUST announce that you are running the checks on the #summit-announce slack channel before execution. 
+
+The scripts can all be found under the Standard scripts panel. 
+The order of script execution is nominally:
+
+* auxtel/enable_latiss.py - Can be skipped if LATISS enabled 
+* auxtel/daytime_checkout/latiss_checkout.py
+* auxtel/enable_atcs - Can be skipped if ATCS already enabled
+* auxtel/daytime_checkout/atpneumatics_checkout.py
+* auxtel/daytime_checkout/telescope_and_dome_checkout.py
+* auxtel/daytime_checkout/slew_and_take_image_checkout.py
+* auxtel/prepare_for/flats - Can be substituted for other prepare_for scripts to leave AuxTel in different state
+
+*auxtel/enable_latiss.py*
+===============
+
+Bring LATISS to an enabled state to prepare for checkout. 
+
+*auxtel/daytime_checkout/latiss_checkout.py*
+=================
+
+This script will take a bias and engineering frame with LATISS and ensure that they are ingested by the OODS.
+
+Check that the bias pops up in the monitor (https://roundtable.lsst.codes/rubintv/summit/auxtel/monitor_current)
+If it fails, let Patrick or Merlin know.
+
+You can check that it ingested at USDF with this link: ******* <b>Add when available<b>
+
+*auxtel/daytime_checkout/enable_atcs.py*
+=============
+
+Bring the ATCS into an enabled state to prepare for checkout. 
+
+*Standrd script - auxtel/daytime_checkout/atpneumatics_checkout.py*
```

Comment:
I think the "Standard Script" is not necessary as you have already mentioned it above.